### ISSUE TITLE
fix markdown formatting on /install/server-infrastructure/

### ISSUE THREAD

**Issue by ElPaisano** [ipfs/ipfs-docs#1356]
Looking through https://docs.ipfs.tech/install/server-infrastructure/, I see a few things that I'd like to propose that we update.

1. ~~Opening para https://docs.ipfs.tech/install/server-infrastructure/#server-infrastructure to better convey why someone would use IPFS cluster and what it's benefits are. Propose adding a '## Features' subheader below the opening para. Proposed copy here https://github.com/ipfs/ipfs-docs/issues/1356#issuecomment-1341494456. Looking to align more closely with https://ipfscluster.io/~~ out of scope for this issue. Tracked by https://github.com/ipfs/ipfs-docs/issues/1370
2. The TIP box below the embedded video - the link is formatted improperly. Also, RE the content in this tip box, as a noob IPFS user, it's not clear to me why this info is here. We should rewrite the copy to clarify, move it elsewhere, etc.
3. Header structure - [Prerequisites](https://docs.ipfs.tech/install/server-infrastructure/#prerequisites) should be the same level as [Create a local cluster](https://docs.ipfs.tech/install/server-infrastructure/#create-a-local-cluster) so that it is visible in the page sub menu and and so that the reader is more aware there are prereqs
4. Procedure copy and formatting - for example, step 4 calls out possible errors. These should probably be in a tip box or something like that, as the formatting is currently confusing. Looking through the procedure steps, I notice that some sort of markdown formatting issue is causing step numbers to be repeated i.e. after step 4, I see steps 1-3 again, and then see a step 1 again. Are these separate sub procedures or a single procedure?

**Comment by ElPaisano** [comment]
Proposed copy for initial para:

```
IPFS Cluster is a distributed application that works as a sidecar to IPFS peers, maintaining a global cluster pinset and intelligently allocating its items to the IPFS peers. This makes it significantly easier to manage multiple IPFS nodes and ensure that data is available across an internal network. IPFS Cluster powers large IPFS storage services like [nft.storage](https://nft.storage/) and [web3.storage](https://web3.storage/). 

## Features 

IPFS Cluster has the following features:

- _Easy to run_: Runs independently from IPFS and the IPFS daemon’s API.
- _Handles replication of millions of pins to hundreds of IPFS daemons:_ Tracks pin lifetime asynchronously, asks IPFS to pin things at a sustainable rate and retries pinning in case of failures.
- _Clever pinning prioritization:_ New pins are prioritized over pin requests that are old or have repeatedly failed to pin.
- _Ingest pins at scale:_ Pins can be added at a rate hundreds of pins per second into the cluster from that moment they are tracked and managed by the cluster peers.
- _Balanced allocation:_ Distributes pins evenly among peers in different groups and subgroups (i.e regions, availability zones), ultimately choosing those with most free storage space available.
- _API and CLI_: Provides a command-line client and a fully featured Cluster HTTP REST API.
_No central server to manage:_ Cluster peers form a distributed network and maintain a global, replicated, conflict-free list of pins.
_Baked-in permissions:_ The embedded permission model supports peers with permissions to change the cluster pinset and peers which store content as instructed but that cannot modify the pinset.
_Name your pins:_ Supports custom replication factors, names and metadata for every pin.
_Multi-peer add:_ Ingests IPFS content to multiple daemons directly.
_CAR import support:_ Directly imports CAR-archived content using custom DAGs.
_IPFS proxy API:_ Cluster peers provide an additional IPFS proxy API that behaves exactly like the IPFS daemon’s API does.
_Integration-ready:_ Cluster peers can be programmatically launched and controlled using Go and Javascript clients for its API.
- Powered by [libp2p](https://libp2p.io/): Built on libp2p, the battle-tested, next-generation p2p networking library used by IPFS, Filecoin and Ethereum V2.
```

**Comment by ElPaisano** [comment]
@TMoMoreau would you be interested in 
a) editing the copy above? My phrasing / formatting is probably lacking here
b) If we agree that this is something worth doing, create a PR draft?

I do think that, at bare minimum, we should fix any MD formatting issues i.e. link formatting and the weird thing happening with the steps

**Comment by ElPaisano** [comment]
Scoping down to "fix markdown formatting on this page"

**Comment by TMoMoreau** [comment]
At least for now, we can come back to the additional content stuffs later.

**Comment by ElPaisano** [comment]
Addressed by https://github.com/ipfs/ipfs-docs/pull/1371
